app/src/main/java/com/example/dynamicdiet/service/DietService.kt

```diff
@@ -17,7 +17,7 @@ class DietService {
      * @param caloriesPerDay The number of calories per day set for the diet
      * @return newly created Diet object
      */
-    suspend fun createDiet(var userId: Int, var name: String, var caloriesPerDay: Int): Diet {
+    suspend fun createDiet( userId: Int,  name: String,  caloriesPerDay: Int) { //removed var
```

Comment:
Removed var and : Diet this allows it to compile

https://kotlinlang.org/docs/coding-conventions.html#class-headers